.gitignore

```diff
@@ -12,3 +12,4 @@ tags
 /build
 docs/_build
 docs/examples
+docs/sg_execution_times.rst
```

Comment:
Dunno if this shows up for you too after running `make html`, but I assume it's meant to be ignored.

Reply:
Could it be the version of Sphinx you are running? This is what I have:

```
Sphinx                        7.3.7
sphinx-gallery                0.18.0
sphinx-markdown-tables        0.0.17
sphinx-rtd-theme              3.0.2
sphinxcontrib-applehelp       2.0.0
sphinxcontrib-devhelp         2.0.0
sphinxcontrib-htmlhelp        2.1.0
sphinxcontrib-jquery          4.1
sphinxcontrib-jsmath          1.0.1
sphinxcontrib-qthelp          2.0.0
sphinxcontrib-serializinghtml 2.0.0
```

Reply:
The dev requirements are open ended, so if you haven't updated them in a while you are likely running a much older version of Sphinx than I am (since I just installed them today).

If you're courageous, I would try `pip install -U -r docs/requirements.txt` and then see if you get the same behavior that I am seeing.

Reply:
Yes, you're right, it gets generated by sphinx-gallery. I think it's fine to just ignore it.